### PR TITLE
V1.725 Remove dependency on edmaps holdings file and add CLI task support

### PR DESCRIPTION
**Pull request by piehld**
- Remove dependency on edmaps holdings file (no longer generating Map Coefficient MTZ files)
- Add CLI support for performing final sanity check for ExDB loading and holdings in `etl.load_ex.DbLoadingWorkflow` task
- Update CI/CD testing to use python 3.10

Jira: https://rcsbpdb.atlassian.net/browse/RO-4344

Goes together with: https://github.com/rcsb/py-rcsb_utils_repository/pull/23

### CODE REVIEW

File: rcsb/db/mongo/PdbxLoader.py [hunk 1]
```diff
@@ -1156,3 +1156,61 @@ def checkValidationDataCount(self, databaseName, collectionName, minValidationCo
...
+            combinedHoldingExpectedCount = 0
+
+            exdbCollections = {
+                "pdbx_core": ["pdbx_core_entry"],
+                "repository_holdings": [
+                    "repository_holdings_combined_entry",
+                    "repository_holdings_current_entry",
+                    "repository_holdings_removed_entry",
+                    "repository_holdings_unreleased_entry",
+                    "repository_holdings_update_entry",
```
Comment (brindakv) [inline rcsb/db/mongo/PdbxLoader.py:1176]: @piehld Where is `repository_holdings_update_entry` used?

File: rcsb/db/processors/RepoHoldingsDataPrep.py [hunk 2]
```diff
@@ -24,6 +24,7 @@
 # 23-Oct-2020 jdw add getHoldingsCombined()
 #  1-Jul-2024 dwp Turn off population of "2fo-fc Map" and "fo-fc Map" content types (DSN6 maps)
 #                 (note that this file is only used when discoveryMode=='local')
+# 16-Oct-2024 dwp Remove usage of EDMAPS holdings file (again, note that this file is only used when discoveryMode=='local')
```
Comment (brindakv) [inline rcsb/db/processors/RepoHoldingsDataPrep.py:27]: @piehld Will this impact the use of `discoveryMode=='local'`?

**Review by brindakv — COMMENTED** [review]
@piehld Please see my comments below.

File: rcsb/db/mongo/PdbxLoader.py (hunk 1 above)
Comment (piehld) [reply rcsb/db/mongo/PdbxLoader.py:1176]: Good question...I don't see where that collection ever actually gets used (I don't see it passed into DW). But this is helpful to keep in mind in the direction of possibly loading deltas.

**Comment by brindakv** [comment]
Thanks @piehld

**Review by brindakv — APPROVED** [review]

**Pull request merged**